docs/source/en/optimization/torch2.0.mdx

```diff
@@ -80,131 +93,78 @@ pip install --upgrade torch torchvision diffusers
 
     Depending on the type of GPU, `compile()` can yield between 2-9% of _additional speed-up_ over the accelerated transformer optimizations. Note, however, that compilation is able to squeeze more performance improvements in more recent GPU architectures such as Ampere (A100, 3090), Ada (4090) and Hopper (H100).
     
```

Comment:
Actually can we change the code here? I would maybe structure it as follows:

1. Explain how torch compile can be used for every pipeline out there by compiling the unet:
=> Just show:
```py
pipe.unet = torch.compile(pipe.unet, mode="reduce-overhead", fullgraph=True)
```

2. Then give examples for the most important models:
- Stable Diffusion
- ControlNet
- IF
=> here we can copy paste our benchmark code